AUTHENTICATION_QUICKSTART.md

```diff
@@ -0,0 +1,236 @@
+# Authentication Quick Start Guide
+
+This guide will help you quickly set up and test Webex authentication for the BYOVA Gateway monitoring dashboard.
+
+## Prerequisites
+
+- Python 3.8+ with virtual environment activated
+- Access to [Webex Developer Portal](https://developer.webex.com/)
+- A Webex account
+
+## Step 1: Create a Webex Integration
+
+1. Go to https://developer.webex.com/
+2. Log in with your Webex account
+3. Click **"Create an Integration"** (under "My Apps")
+4. Fill in the form:
+   - **Integration Name**: BYOVA Gateway Monitor (Dev)
```

Comment:
<img width="888" height="151" alt="image" src="https://github.com/user-attachments/assets/37e8b643-2f78-49ed-9569-852cffae58bb" />

This causes an error, let's remove the parens.

```suggestion
   - **Integration Name**: BYOVA Gateway Monitor Dev
```